### PR TITLE
Linter fixes and improvements

### PR DESCRIPTION
**Pull request by wirednkod**
- add  es-linter checks
- Add CI checks for linter

### CODE REVIEW

File: fork-visualizer/.eslintrc.json [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+			"map": [ ["@/*", "./src/*"] ]
+		}
+	}
+  },
+  "rules": {
+    // Stylistic
+    // ---------
+	"no-multiple-empty-lines": "error",
+    "comma-dangle": [
+      "error",
```
Comment (JoshOrndorff) [inline fork-visualizer/.eslintrc.json:18]: Is this formatting valid? And are comments even valid in json?

**Review by JoshOrndorff — APPROVED** [review]
Thank you!

**Comment by JoshOrndorff** [comment]
You may merge this into Kevin's branch whenever you are ready.

File: fork-visualizer/.eslintrc.json (hunk 1 above)
Comment (wirednkod) [reply fork-visualizer/.eslintrc.json:18]: Yes they are :)

**Pull request merged**